magento: update root docker-compose.yml

sameersbn opened the pull request.

petemill commented:
Please could you provide some context as to why the directory structure changed, in case I or others need to update production environment accordingly? Thanks!

petemill commented:
@sameersbn Please could you provide some context as to why the directory structure changed, in case I or others need to update production environment accordingly? Thanks!

sameersbn commented:
Hi. We have removed the VOLUME instruction from the [Dockerfile](https://github.com/bitnami/bitnami-docker-magento/commit/c62a050cc2352533982b249710570798c0005bbb#diff-0d404cdcf172086bfb66b4c4024c21e6L36). For persistence, this enables users to mount a single volume at `/bitnami` instead of three separate volumes. As such we have updated the docker-compose.yml to suggest mounting a single volume. However, you can continue mounting the three volumes as before without any side-effects.
